src/layouts/default/layout.pug

```diff
@@ -6,7 +6,7 @@ html(lang=language)
     meta(name='viewport', content='width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no')
     meta(name='format-detection', content='telephone=no')
 
-    link(rel='stylesheet', href=relativePath + 'css/styles.css')
+    link(rel='stylesheet', href=relativePath + 'css/layouts/default/style.css')
```

Comment:
Not sure this is the simpliest output. But at least it's self explaining so let's keep it for now